ui/src/components/core/input/input-label.ts

```diff
@@ -0,0 +1,24 @@
+import { dripStitches } from '../../../theme/stitches';
+
+const { styled } = dripStitches;
+
+export const StyledInputLabel = styled('label', {

```

Comment:
Move this also to `input.styled.ts` so we have a single file for input stylings.